Azure/sonic-buildimage
# [sonic-sairedis] Submodule update

Signed-off-by: Myron Sosyak <msosyak@barefootnetworks.com>



**- What I did**
Updated sonic-sairedis submodule with following changes:

- acquire lock before accessing shared redis context ([#489](https://github.com/Azure/sonic-sairedis/pull/489))
- Fix bug: Restore sai log levels from LOGLEVEL_DB after saiDiscover ([#491](https://github.com/Azure/sonic-sairedis/pull/491))
- Fixed p4 profile change for BFN platform ([#492](https://github.com/Azure/sonic-sairedis/pull/492)) 

**- How I did it**
Updated sonic-sairedis submodule

**- How to verify it**
- Deploy image and verify all is up and running
- Run ansbile tests

**- Description for the changelog**

[sonic-sairedis] Submodule updated
